compiler/luci/import/include/luci/Import/GraphBuilderMulti.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_IMPORT_GRAPH_BUILDER_MULTI_H__
+#define __LUCI_IMPORT_GRAPH_BUILDER_MULTI_H__
+
+#include "GraphBuilderContext.h"
+#include "GraphBuilderBase.h"
+
+#include <mio/circle/schema_generated.h>
+
+namespace luci
+{
+
+/**
+ * @brief Base of general multiple outputs graph builder(e.g., CircleIfGraphBuilder)
+ */
+class GraphBuilderMulti : public GraphBuilderBase
```

Comment:
As `Multi` is a prefix, not noun, it seems little bit weird.
How about `MultipleOutputsGraphBuilder`, `MultiOutputsGraphBuilder` or `MultiOutGraphBuilder`?

Reply:
I'd like to keep `GraphBuilder` as prefix of this series... so, is `GraphBuilderMultiOutput` OK?

Reply:
`GraphBuilderMultiOutput` looks fine to me :D